### PR TITLE
feat: Support --random and --seed flags

### LINKED ISSUE #137: Specify config data for Jasmine
Apologies if I'm missing something stupid, but I'm having difficulty figuring out how/where to specify config data for Jasmine runs (which normally goes in a config file specified in Jasmine settings or through the loaded library). Specifically, I'm looking to set `random:true` through jasmine/karma config.

Thanks in advance,
Zack

### LINKED ISSUE #145: Support --random and --seed flags
https://github.com/jasmine/jasmine/pull/927
https://jasmine.github.io/2.4/node.html#section-CLI_Options

### PR DESCRIPTION
**Pull request by maksimr**
close #145
close #137

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,23 @@ module.exports = function(config) {
...
 
+If you want to pass configuration options directly to jasmine you can do this in the following way
+
+```js
+module.exports = function(config) {
+  config.set({
+    ...
+    client: {
+      jasmine: {
+          random: true,
```
Comment (dignifiedquire) [inline README.md]: inconsistent indentation

File: src/adapter.js [hunk 2]
```diff
@@ -319,11 +321,24 @@ var createSpecFilter = function (config, jasmineEnv) {
...
+    setOption(jasmineConfig.stopOnFailure, jasmineEnv.throwOnExpectationFailure)
+    setOption(jasmineConfig.seed, jasmineEnv.seed)
+    setOption(jasmineConfig.random, jasmineEnv.randomizeTests)
+
     jasmineEnv.addReporter(new KarmaReporter(karma, jasmineEnv))
     jasmineEnv.execute()
   }
+
+  function setOption (option, set) {
+    if (option !== undefined) {
```
Comment (dignifiedquire) [inline src/adapter.js]: Better to use `!= null`, so it checks for both `undefined` and `null`. Or you could use `lodash.merge` to just merge the config objects.

Comment (maksimr) [inline src/adapter.js]: done

**Comment by dignifiedquire** [comment]
Thanks :octocat:

**Comment by deiga** [comment]
@dignifiedquire @maksimr Is there any way for karma to report the seed? I can only find custom reporters with this functionality

**Comment by maksimr** [comment]
@deiga only custom reporter but you could create inline reporter without any additional dependencies

[Example](https://gist.github.com/maksimr/aa8444db54b4897df1ed0d75007a4840)

Thanks!

**Pull request merged**